HealthcareSystem/PatientWebApp/Controllers/FeedbackController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Backend.Service.NotificationSurveyAndFeedback;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Model.Users;
+using PatientWebApp.Adapters;
+using PatientWebApp.DTOs;
+
+namespace PatientWebApp.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class FeedbackController : ControllerBase
+    {
+        private readonly IFeedbackService _feedbackService;
+        public FeedbackController(IFeedbackService feedbackService)
+        {
+            _feedbackService = feedbackService;
+        }
+
+        [HttpPost("addFeedback")]
```

Comment:
This isn't how [REST resources are usually named](https://restfulapi.net/resource-naming/), so I'd switch to a more conventional naming scheme when the rest of the methods are added.